services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_scheduler_utils.py

```diff
@@ -98,7 +98,9 @@ async def discover_running_services(scheduler: "Scheduler") -> None:  # type: ig
     logger.info("The following services need to be observed: %s", services_to_observe)
 
     for scheduler_data in services_to_observe:
-        await scheduler._add_service(scheduler_data)  # pylint: disable=protected-access
+        await scheduler._add_service(  # pylint: disable=protected-access  # noqa: SLF001
```

Comment:
again I don't really understand the motivation behind having a private method and still using it??

Reply:
I agree